test/connectionStore.test.ts

```diff
@@ -576,5 +576,69 @@ suite('ConnectionStore tests', () => {
         });
     });
 
+    test('addSavedPassword not lookup password if password already set', (done) => {
+        // Given a profile with emptyPasswordInput set
+        let passwordAlreadySetProfile = Object.assign(new ConnectionProfile(), {
+            profileName: 'defaultNamedProfile',
+            server: 'namedServer',
+            database: 'bcd',
+            authenticationType: utils.authTypeToString(interfaces.AuthenticationTypes.SqlLogin),
+            user: 'cde',
+            password: 'asdf!@#$'
```

Comment:
FYI, everywhere we add the text 'password" we'll get a TSA credential scan bug.  if we do need to include a password I would recommend using the sting "password" which is more obviously not a real password